test/e2e/suite-plugins/referer-restriction.go

```diff
@@ -101,6 +101,125 @@ spec:
 		resp.Body().Contains("Your referer host is not allowed")
 	})
 
+	ginkgo.It("referer-restriction plugin configuration blacklist list", func() {
```

Comment:
```suggestion
	ginkgo.It("referer-restriction plugin configuration deny list", func() {
```

Reply:
black and white will make people think of skin color, how about using deny list instead of it?

Reply:
OK~